src/EDF.jl

```diff
@@ -154,9 +158,60 @@ mutable struct Signal
     digital_max::Float32
     prefilter::String
     n_samples::Int16
+end
+
+"""
+    EDF.AnnotationHeader
+
+Type representing the header record for an `AnnotationList`
+
+# Fields
+
+* `n_samples` (`Int16`): The number of samples in a single data record
+"""
+struct AnnotationHeader
```

Comment:
> Type representing the header record for an `AnnotationList`

Would `AnnotationListHeader` then be a better name?

Reply:
Strong agree.

Reply:
Fixed in https://github.com/beacon-biosignals/EDF.jl/pull/23/commits/61e14cba29ecf94cdd897501e71e5fb8daecc48f